openshift-install-powervs

```diff
@@ -1138,7 +1139,7 @@ function setup_occli() {
 
   EXT_PATH=$(which oc 2> /dev/null || true)
 
-  if [[ -f $OC_PATH && $($OC_PATH version --client | sed 's/.*version.* //I')  == *"${RELEASE_VER}"* ]]; then
+  if [[ -f $OC_PATH && $($OC_PATH version --client | sed 's/.*version.* //')  == *"${RELEASE_VER}"* ]]; then
```

Comment:
Its `Version` if you remove the ignore case.

Reply:
Resolved.